### PR TITLE
Fix for the glibc issue on RHEL/Centos

### PR DESCRIPTION
**Pull request by Psychomantis71**
Not the most elegant solution, but does the trick for people using RHEL/Centos

I added 3 new parameters and edited the readme file, in short:
-glibc-custom (if you are using custom glibc, true/false, default false)
-glibc-loc (path to the glibc ld.so file)
-glibc-lib-loc (path to the glibc lib folder)

### CODE REVIEW

File: src/factorio_server.go [hunk 1]
```diff
@@ -140,12 +146,27 @@ func (f *FactorioServer) Run() error {
...
 	}
 
-	args := []string{
-		"--bind", (f.BindIP),
-		"--port", strconv.Itoa(f.Port),
-		"--server-settings", filepath.Join(config.FactorioConfigDir, "server-settings.json"),
-		"--rcon-port", strconv.Itoa(config.FactorioRconPort),
-		"--rcon-password", config.FactorioRconPass,
+	args := []string{}
+	if config.glibcCustom == "true" {
```
Comment (mroote) [inline src/factorio_server.go]: Instead of repeating the args here it would be a bit cleaner just to append the `--library-path` parameter to the args slice if it's specified.

**Comment by mroote** [comment]
Thanks for taking a look at this! I left a small comment but otherwise looks good.

File: src/factorio_server.go (hunk 1 above)
Comment (Psychomantis71) [reply src/factorio_server.go]: You are right, I'm an idiot, thanks! :)

**Comment by Psychomantis71** [comment]
@mroote I took your advice and made it so that the args will be appended. Also added a new parameter, -"autostart", it will start the Factorio server together with FSM when set to true (But I need to do some more testing on that one to be sure that there are no weird edge case bugs).

Cheers!

**Comment by mroote** [comment]
Nice thanks! I think for this PR we should just keep it to the glib fixes just to get them merged in quicker. The autostart sounds great I'd like to add it for sure just don't want to delay the bugfix.

File: src/factorio_server.go [hunk 2]
```diff
@@ -156,7 +194,13 @@ func (f *FactorioServer) Run() error {
 
 	log.Println("Starting server with command: ", config.FactorioBinary, args)
```
Comment (mroote) [inline src/factorio_server.go]: This logline can be removed now.

**Comment by Psychomantis71** [comment]
@mroote  Sure thing, I'll remove the extra log line and autostart feature from my master brach, create a dev branch where I'll add the autostart option and other ideas I have, so you can just merge the master branch.

**Comment by mroote** [comment]
You can specify settings in the conf.json file so you don't have to provide the flags every time.

**Comment by Psychomantis71** [comment]
Oh I forgot about that, sorry! I'll do the stuff from above after work, if there are any other problems you may need a hand with let me know.

Cheers!

File: README.md [hunk 3]
```diff
@@ -59,11 +59,25 @@ Usage of ./factorio-server-manager:
...
   -port string
     	Specify a port for the server. (default "8080")
-
+  -glibc-custom string 
+        Specify if custom glibc is used (default false) [true/false]
+  -glibc-loc string
+        Path to the glibc ld.so file (default "/opt/glibc-2.18/lib/ld-2.18.so")
+  -glibc-lib-loc
+        Path to the glibc lib folder (default "/opt/glibc-2.18/lib")
+  -autostart
```
Comment (mroote) [inline README.md]: I'll just remove this param for now.

**Comment by mroote** [comment]
Thanks for the fix!
